tests/ClientFactoryTest.php

```diff
@@ -58,16 +58,22 @@ public function testGetClient(): void
 
         $clientFactory = new ClientFactory(
             'https://dummy',
-            ['handler' => $stack, 'logger' => $logger, 'userAgent' => 'test agent']
+            ['handler' => $stack, 'logger' => $logger, 'backoffMaxTries' => 3, 'userAgent' => 'test agent']
```

Comment:
Koukam tu na ty options a krome toho, ze je teda celkem mordor co delaji ty options v konstruktoru :smile: (musel jsem fakt kouknout do projektu, abych pochopil, ze jsou to options pro nejakeho interniho SAPI clienta), prijde mi, ze treba `userAgent` by vlastne mohl byt konfigurovany jen jednou a vsude stejny, ne? Protoze reprezentuje toho, kdo pouziva toho clienta.

Vim ze to tam uz takle bylo, jen me to tu ted konecne trklo do oka, kdyz se to objevilo v testu.

Reply:
však je definovaný jednou ne? Jako tohle nastavíš někde v services a dál to neřešíš. Mordor je tady v tom, že jsem to udělal povinný, protože to zapomínáme nastavovat